stubs/braintree/braintree/unknown_payment_method.pyi

```diff
@@ -1,4 +1,6 @@
+from typing import Literal
+
 from braintree.resource import Resource
 
 class UnknownPaymentMethod(Resource):
-    def image_url(self): ...
+    def image_url(self) -> Literal["https://assets.braintreegateway.com/payment_method_logo/unknown.png"]: ...
```

Comment:
Similar here: The exact value shouldn't matter to the caller and may change in the future.

```suggestion
    def image_url(self) -> str: ...
```